Rakefile

```diff
@@ -5,7 +5,8 @@ require 'rspec/core/rake_task'
 require 'rubocop/rake_task'
 require 'open-uri'
 
-LIB_PG_QUERY_TAG = '16-5.0.0'.freeze
+#LIB_PG_QUERY_TAG = '16-5.0.0'.freeze
+LIB_PG_QUERY_TAG = 'windows-support'.freeze
```

Comment:
This is temporary and will be removed once we have a tagged libpg_query release (same for the commented out if condition below).